vtxmenu.ini

```diff
@@ -64,7 +64,7 @@ Submenu2=VTX MENU
 [ISP]
 Option1=EXPOSURE:0-50:cli -g .isp.exposure:cli -s .isp.exposure {}
 Option2=ANTIFLICKER:DISABLED,50,60:cli -g .isp.antiFlicker | tr a-z A-Z:cli -s .isp.antiFlicker $(echo {} | tr A-Z a-z)
-Option2=IQSERVER:FALSE,TRUE:cli -g .isp.iqServer | tr a-z A-Z:cli -s .isp.iqServer $(echo {} | tr A-Z a-z)
+Option3=SENSOR FILE:/ETC/SENSORS/IMX415_MILOS10.BIN,/ETC/SENSORS/IMX415_MILOS11.BIN:cli -g .isp.sensorConfig | tr a-z A-Z:cli -s .isp.sensorConfig $(echo {} | tr A-Z a-z)
```

Comment:
This might be a problem. When the read value does not match any of element in the list the first one is used.
So when by default the milos bin are not in the firmware, this will lead to a sensor config /etc/sensors/imx415_milos10.bin after the first save in the ISP section even when it's not there.
Does this work with majestic ?
There are also some shenanigans going on in datalink tweaksys.
Not sure how to handle this in a generalized way.

Reply:
Here is the defaulting: https://github.com/OpenIPC/msposd/blob/main/osd/util/ini_parser.c#L247
In this loop the actual value is set https://github.com/OpenIPC/msposd/blob/main/osd/util/ini_parser.c#L277-L282
When `strcmp(output, value_list[i]) == 0` is never true the default stays.

Reply:
The sensorConfig options is more like an example of how to import and then select an available bin file directly from the VTX menu.
If the majestic doesn’t find the bin file that the user chose, it will automatically use the default bin file based on the camera sensor. 
We are planning to use the Milos calibration files as default for imx415 and imx335 once we have an acceptable result. However the sensorConfig will give flexibility to the user to select a different calibration file based on their preferences or weather conditions.